version/version.go

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 Authors of KubeArmor
+
+package version
+
+import (
+	"context"
+	"fmt"
+	"runtime"
+
+	"github.com/kubearmor/kubearmor-client/k8s"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var version string
+
+func PrintVersion(c *k8s.Client) error {
+	fmt.Printf("karmor version %s %s/%s\n", version, runtime.GOOS, runtime.GOARCH)
```

Comment:
we need build-date as well as part of this.

Reply:
Sure I will add that but any particular reasoning for that? I have usually seen only versions and runtime info.